sdk/tests/conformance2/rendering/blitframebuffer-test.html

```diff
@@ -52,15 +52,56 @@
     testFailed("WebGL context does not exist");
 } else {
     testPassed("WebGL context exists");
+    blit_framebuffer_repeated();
     blit_framebuffer_feedback_loop();
     blit_framebuffer_multisampling_srgb();
 }
 
+function blit_framebuffer_repeated() {
+    debug("");
+    debug("This test verifies repeated calls to blitFramebuffer.");
+
+    // Create offscreen fbo and its color attachment.
+    var tex_2d = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, tex_2d);
+    gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA8, width, height, 0, gl.RGBA, gl.UNSIGNED_BYTE, null);
```

Comment:
Slight preference for texStorage in webgl2-only code.

Reply:
Fixing in #2569.